### PR TITLE
feat: update smoke tests to DAPI v0.19

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Smoke tests were failing against Palinka network after it was upgraded to Dash Platform `0.19-dev`. This PR attempts to fix the following test failures:
```
masternode-5
        ✓ should respond data from Core
Response deserialization failed: invalid wire type 6 at offset 131
Response deserialization failed: invalid wire type 6 at offset 131
Response deserialization failed: invalid wire type 6 at offset 131
Response deserialization failed: invalid wire type 6 at offset 131
Response deserialization failed: invalid wire type 6 at offset 131
Response deserialization failed: invalid wire type 6 at offset 131
        32) should respond data from Core using GRPC
        ✓ should respond data from Platform
        ✓ should respond data from TxFilterStream GRPC service (184ms)
32) Smoke
       DAPI
         masternode-5
           should respond data from Core using GRPC:
     MaxRetriesReachedError: Max retries reached: 13 INTERNAL: Failed to parse server response
      at GrpcTransport.request (node_modules/@dashevo/dapi-client/lib/transport/GrpcTransport.js:88:15)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```
pip was also throwing warnings about being invoked in a deprecated manner.

## What was done?

- Manually update `@dashevo/dapi-client` to `v0.19-dev` and run `npm update`
- Change pip invocation syntax from `pip3` to `python3 -m pip`

## How Has This Been Tested?

By building and running smoke tests against Palinka

## Breaking Changes

None known

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by strophy** [comment]
Smoke tests are now failing with:
```
  39) Smoke
       DAPI
         masternode-5
           should respond data from Core using GRPC:
     AssertionError: expected { Object (version, time, ...) } to have property 'coreVersion'
      at Context.it (test/smoke/dapi.js:55:34)
```

**Review by shumkov — APPROVED** [review]
👍

**Comment by shumkov** [comment]
@strophy we changed the format of getStatus endpoint response so you need to update tests https://github.com/dashevo/dapi-grpc/blob/v0.19-dev/protos/core/v0/core.proto#L19-L86

**Comment by strophy** [comment]
Thanks for that! Tests now fail with:
```
  39) Smoke
       DAPI
         masternode-6
           should respond data from Core using GRPC:
     AssertionError: expected { Object (version, time, ...) } to have property 'sync_progress'
      at Context.it (test/smoke/dapi.js:58:34)
```

Commenting out `expect(result).to.have.a.property('sync_progress');` fixes this. I see the `sync_progress` value is also contained in the `Masternode` and `Chain` messages, is this a mistake in the smoke tests or in `dapi-grpc`?

File: .env [hunk 1]
```diff
@@ -0,0 +1,29 @@
+TERRAFORM_S3_BUCKET="dash-networks-deploy-state"
```
Comment (shumkov) [inline .env:1]: You shouldn't commit your `.env` :)

File: ansible/deploy.yml [hunk 2]
```diff
@@ -2,68 +2,68 @@
```
Comment (shumkov) [inline ansible/deploy.yml:2]: Please revert these changes.

File: ansible/group_vars/all [hunk 3]
```diff
@@ -126,6 +126,6 @@ mn_evo_services_path: '{{ dashd_home }}/{{ mn_evo_services_compose_project_name
 # Elastic
```
Comment (shumkov) [inline ansible/group_vars/all:126]: Revert

File: bin/dash-network [hunk 4]
```diff
@@ -38,8 +38,8 @@ fi
 docker run -ti --rm \
```
Comment (shumkov) [inline bin/dash-network:38]: Revert

File: networks [hunk 5]
```diff
@@ -0,0 +1 @@
+../dash-network-configs/devnet-palinka/
```
Comment (shumkov) [inline networks:1]: Revert

File: networks/.gitignore [hunk 6]
```diff
@@ -1,2 +0,0 @@
-*
```
Comment (shumkov) [inline networks/.gitignore:1]: Revert

**Review by shumkov — APPROVED** [review]
👍
